### ISSUE TITLE
Promote Open Source Development among college students through DSC

### ISSUE THREAD

**Issue by ishank-dev** [dsc-rit/DSC-Website-New#53]
**Abstract**
We all know that Open Source Development is very important, its not just related to coding it also teaches us how to communicate your thoughts in a community and enables the users to read large code base written by some other developer  (which eventually we all have to do when we join companies or startups). I realised I got to learn a lot after contributing to open source and I really wished I had some basic open source development experience right in the initial days of college.
**How can we promote open source?**
To understand that lets see the current scenario
**Current Scenario**
Student builds a project, makes a beautiful readme to enable others to come and contribute (which unfortunately never happens) and eventually the repository ends up as trash or isn't built completely. So I feel a **lot of projects are going to waste and are rotting on giithub** either because of lack of team or they got stuck on something that they couldn't implement.
**Solution**
I propose **two solution** to it
1. Enable others to create a repository on the organisation they can request invite, this way others can see what all projects needs attention and based on their tech stack they can just come here and solve good first issues or start contributing.
2. Make a very minimal platform web or app where student can come and list their project ideas and fill in the following 
- **Project Type** : Open Source Existing Project/ Open Source Project Idea/ Feature Request to existing project/ 
- **Current Team size** : 
- **Summary**: A short summary of what the project is about
- **Description of project**: Solve issue/Improve Documentation/Add xyz feature etc.
- **Tech Stack** : Javascript/python/Angular/Vue/React etc
- **Project URL** : URL of the project
The second approach will be helpful if the student does not want to reveal his project source code on the organisation and can just come to our platform and list his project.
To summarise it would be just a small platform to broadcast your project to the college students and gain contributors
**Target Audience**
Student with projects who want to gain contributors.
Student who wants to improve their knowledge of open source contribution and enhance their knowledge of git.

**TLDR**
This platform will allow you to broadcast your existing project or idea/ lets other see your project and contribute.

Please leave opinion on the above idea and what all anticipated challenges we can have etc. If the idea sounds trash then tell that as well, we can alternatively go with a simple approach of doing the same thing on slack.
